bundle/language_en.properties

```
@@ -662,6 +664,52 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.inventory.acanceledinventorycannotbemodified.msg                                                 = A canceled inventory can not be modified
+angal.inventory.allproduct.radio                                                                       = All products
+angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg                          = Cannot create another inventory with other in progress in this ward.
+angal.inventory.date.label                                                                             = Inventory's date
+angal.inventory.edit.title                                                                             = Edit an inventory
+angal.inventory.error.msg                                                                              = Error during  some products's recording
+angal.inventory.lotcode.col                                                                            = Lot's code
+angal.inventory.managementward.title                                                                   = Manage Ward Inventory
+angal.inventory.medical.col                                                                            = Medical
+angal.inventory.mustenterareference.msg                                                                = You must enter a reference.
+angal.inventory.nextarrow.btn                                                                          = >
+angal.inventory.noproduct.msg                                                                          = Product is not found.
+angal.inventory.noproductfound.msg                                                                     = No medical found
+angal.inventory.noteditable.msg                                                                            = A validated inventory can not be modified
```

```suggestion
angal.inventory.noteditable.msg                                                                        = A validated inventory cannot be modified.
```

In this diff `can not` is used twice and `cannot` is used 10 times.

Also messages should be statements that end in a period.